src/test/features/dashboard/routes/index.ts

```diff
@@ -77,7 +95,317 @@ describe('Dashboard page', () => {
+              const payBySetDateClaim = {
+                ...fullAdmissionClaim,
+                response: { ...fullAdmissionClaim.response, ...basePayBySetDateData }
+              }
+              resolveRetrieveByDefendantId('000MC001', '1', payBySetDateClaim)
+
+              await request(app)
+                .get(Paths.dashboardPage.uri)
+                .set('Cookie', `${cookieName}=ABC`)
+                .expect(res => expect(res).to.be.successful.withText('You’ve admitted all of the claim and offered to pay the full amount by 31 December 2050.'))
+            })
+
+            it('should show pay immediately statuses when the claim is marked as pay by instalments', async () => {
+              const payByInstalmentsDateClaim = {
+                ...fullAdmissionClaim,
+                response: { ...fullAdmissionClaim.response, ...basePayByInstalmentsData }
+              }
+              resolveRetrieveByDefendantId('000MC001', '1', payByInstalmentsDateClaim)
+
+              await request(app)
+                .get(Paths.dashboardPage.uri)
+                .set('Cookie', `${cookieName}=ABC`)
+                .expect(res => expect(res).to.be.successful.withText('You’ve admitted all of the claim and offered to pay the full amount in instalments.'))
+            })
+          })
+          context('when the claim is in partial admission', () => {
+            const partAdmissionClaim = {
+              ...sampleClaimObj,
+              responseDeadline: MomentFactory.currentDate().add(1, 'days'),
+              response: {
+                ...baseResponseData,
+                ...basePartialAdmissionData,
+                amount: 30
+              }
+            }
+            const payImmediatelyClaim = {
+              ...partAdmissionClaim,
+              response: { ...partAdmissionClaim.response, ...basePayImmediatelyData }
+            }
+            const acceptedClaim = {
+              ...payImmediatelyClaim,
+              claimantResponse: baseAcceptationClaimantResponseData
+            }
+            it('should show pay immediately statuses when the claim is marked as pay immediately', async () => {
+              resolveRetrieveByDefendantId('000MC001', '1', payImmediatelyClaim)
+
+              await request(app)
+                .get(Paths.dashboardPage.uri)
+                .set('Cookie', `${cookieName}=ABC`)
+                .expect(res => expect(res).to.be.successful.withText('You’ve admitted part of the claim.'))
+            })
+
+            it('should show pay immediately statuses when the claim is marked as pay immediately and the offer is accepted', async () => {
+              resolveRetrieveByDefendantId('000MC001', '1', acceptedClaim)
+
+              await request(app)
+                .get(Paths.dashboardPage.uri)
+                .set('Cookie', `${cookieName}=ABC`)
+                .expect(res => expect(res).to.be.successful.withText('John Smith accepted your admission of 30'))
```

Comment:
Is this exposing a UI bug? Should have the £ prefix, surely?

Reply:
yes, it is a bug I found and I have a fix ready for it :)

Reply:
I can actually put the fix in right now, now that I think of it. It was in another PR, but that might take a long(er) time to get merged

Reply:
Committed the fix :D _scope creep 😆 _